### PR TITLE
EZP-31343: Added sorting asc/desc to items of udw

### PR DESCRIPTION
**Pull request by OstafinL**
Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31343
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by OstafinL** [comment]
sorting
sorting_revers

**Comment by dew326** [comment]
ping @inakijv

File: Resources/public/scss/modules/universal-discovery/_sort.switcher.scss [hunk 1]
```diff
@@ -8,4 +8,31 @@
...
     }
+
+
+    .c-menu-button {
+        position: relative;
+        padding: 0 calculateRem(18px);
+
+        &--selected::after {
+            content: '';
+            width: 0px;
```
Comment (GrabowskiM) [inline Resources/public/scss/modules/universal-discovery/_sort.switcher.scss]: px

File: src/modules/universal-discovery/components/sort-switcher/sort.switcher.js [hunk 2]
```diff
@@ -4,10 +4,14 @@ import PropTypes from 'prop-types';
...
 
 import { createCssClassNames } from '../../../common/helpers/css.class.names';
-import { SortingContext, SORTING_OPTIONS } from '../../universal.discovery.module';
+import { SortingContext, SortOrderContext, SORTING_OPTIONS } from '../../universal.discovery.module';
 
 const SortSwitcher = ({ isDisabled }) => {
     const [sorting, setSorting] = useContext(SortingContext);
+    const [sortOrder, setSortOrder] = useContext(SortOrderContext);
+
+    const ascendingOrder = 'ascending';
```
Comment (GrabowskiM) [inline src/modules/universal-discovery/components/sort-switcher/sort.switcher.js]: personally I would move this kind of consts outside of component and write them in uppercase like `const ASCENDING_ORDER = 'ascending'` to distinct them from other variables, question for @dew326 what he thinks about it

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

File: Resources/public/scss/modules/universal-discovery/_sort.switcher.scss [hunk 3]
```diff
@@ -8,4 +8,31 @@
     &--disabled {
         opacity: 0.3;
     }
+
```
Comment (GrabowskiM) [inline Resources/public/scss/modules/universal-discovery/_sort.switcher.scss:11]: unnecessary empty line

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

File: src/modules/universal-discovery/components/sort-switcher/sort.switcher.js [hunk 4]
```diff
@@ -4,10 +4,15 @@ import PropTypes from 'prop-types';
...
-import { SortingContext, SORTING_OPTIONS } from '../../universal.discovery.module';
+import { SortingContext, SortOrderContext, SORTING_OPTIONS } from '../../universal.discovery.module';
+
+const ASCENDING_ORDER = 'ascending';
+const DESCENDING_ORDER = 'descending';
 
 const SortSwitcher = ({ isDisabled }) => {
     const [sorting, setSorting] = useContext(SortingContext);
+    const [sortOrder, setSortOrder] = useContext(SortOrderContext);
+
```
Comment (dew326) [inline src/modules/universal-discovery/components/sort-switcher/sort.switcher.js]:
```suggestion
```

**Review by dew326 — CHANGES_REQUESTED** [review]

**Comment by dew326** [comment]
I'm not sure if the left padding in the button should be that big. For me it looks not ok, but this is question to @inakijv

**Review by dew326 — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Comment by inakijv** [comment]
@dew326 I checked with @lucasOsti the different alternatives and his proposal is the most consistent with other sorting options we have in our UI.

**Review by inakijv — APPROVED** [review]

**Review by bogusez — APPROVED** [review]

**Pull request merged**